### DISCUSSION TITLE
add extra columns to the pg-report or gg-report

### DISCUSSION THREAD

**Discussion by DaliJarboui** [vdemichev/DiaNN#495] [Q&A]
Hi, 
is there any way to add some extra columns to the report-pg, or -gg matrix, such as the number of peptides (or unique peptides, from each experiment used for quantification). 
thanks for your help.

**Comment by vdemichev** [comment]
All peptides annotated to a protein group in the main report with run-specific q-value < 1% are used to quantify this protein group. Only unique peptides are used to calculate Genes.MaxLFQ.Unique.

**Reply by diabeginner01** [reply]
Isn't it top 3 peptides/precursors that are used for quantification of the protein group?

**Reply by vdemichev** [reply]
No, it's MaxLFQ now

**Comment by DaliJarboui** [comment]
What I mean is an extra output column in the report files for pg or gg where we can see the number of peptides used for the quantification (gene1: N unique peptides in file 1/ file 2 etc....). 
the report.pg (or gg).tsv contains only MaxLFQ values without information on the number of unique peptides used for the quantification.

**Reply by vdemichev** [reply]
Yes, I agree this would be helpful. Not sure when will be implemented though. As is, can easily get this info with a simple R or Python script though.
